tests/issue_85_module.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
```

Comment:
As mentioned in the general comment: the test could probably also be valid without daclasses, thus working on python 3.6 and even 3.5

Reply:
3.6 has reached end of life, but I'm happy to make the changes. I don't see a way to make the tests work on python 2, but will update to skip it as you suggest. Setting `__wrapped__`/`__signature__` will still work on python 2, though it won't do anything.

I'll add a comment when this is ready for more review.